test/unit/create-contentful-api-test.js

```diff
@@ -352,3 +352,65 @@ test('CDA call sync fails', (t) => {
     teardown()
   })
 })
+
+test('Given json should be parsed correctly as a collection of entries', (t) => {
+  const client = contentful.createClient({accessToken: 'blablabla', space: 'bla'})
```

Comment:
then we should not rewire it, but use the same way of initializing the object?

I just think, that dropping in a new loaded file just for one test case, is not as clean as it should be.

The other test cases perform checks on methods that are right next to the new method we added e.g. ' getEntries'.

Maybe we can use this

```
const api = createContentfulApi({
  http: {
    get: getStub.returns(promise)
  },
  shouldLinksResolve: shouldLinksResolve
})
```

and use it without stubbing?

Reaching out to `../../` just feels like mixing the concept of this file up.